docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: '3.7'
+
+services:
+  mysql:
+    image: mysql:8.0
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+    volumes:
+      - mysql:/var/lib/mysql
+
+  postgres:
+    image: postgres:13.2
+    environment:
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: password
```

Comment:
In a personal project I'm using `POSTGRES_HOST_AUTH_METHOD: trust` to skip `POSTGRES_PASSWORD` and to keep the default Rails database configuration (but for the user I'm doing the same as you).

Reply:
In this setup we need both mysql and postgres to match their credentials to work with the generated `database.yml` file.